### DISCUSSION TITLE
learn/training/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#341] [Website Discussions]
# learn/training/

We're big fans of Ripple Training.

https://fcp.cafe/learn/training/

**Comment by IgorGeneralov** [comment]
Textbook in Russian:
Final Cut Pro X. Deep Diving. 1 part.

200 pages
180 drawings and diagrams
In this part 1, the issues of Ideology, Architecture and Program Interface are discussed in detail.

Original: https://shop.generalov.net/books/final-cit-pro-x-russia
Apple books: https://books.apple.com/us/book/final-cut-pro-x-полное-погружение-часть-1/id1485730065
